pkg/config/config_template.yaml

```diff
@@ -1,424 +1,781 @@
-#   forwarded locally to a proxy. It is highly recommended to then handle the SSL encryption
-#   on the proxy side.
-#   logs_no_ssl: false
+#  secret_backend_output_max_size: 1024
+
+## @param secret_backend_timeout - integer - optional - default: 5
+## The timeout to execute the command in second
 #
-#   Global processing rules that are applied to all the logs. The available rules are
-#   "exclude_at_match", "include_at_match" and "mask_sequences". More information in the documentation:
-#   https://docs.datadoghq.com/logs/log_collection/?tab=tailexistingfiles#advanced-log-collection-functions
-#   processing_rules:
-#     - rule1_arg1
-#       rule1_arg2
-#     - rule2_arg1
-#       rule2_arg2
+#  secret_backend_timeout: 5
+
+{{ end -}}
+{{- if .LogsAgent }}
+
+#########################################
+## Logs Collection Agent Configuration ##
+#########################################
+
+## @param logs_enabled - boolean - optional - default: false
+## Enable Datadog Agent logs collection by setting logs_enabled to true.
 #
-#   By default, logs are sent to port 10516 (for the US site), use this parameter
-#   to force the agent to send logs in TCP to port 443 (default is false)
-#   use_port_443: false
+#  logs_enabled: false
+
+## @param logs_config - custom object - optional
+## Enter specific configurations for your Log collection.
+## Uncomment this parameter and the one below to enable them.
+## See https://docs.datadoghq.com/logs/log_collection/
 #
+#  logs_config:
+
+  ## @param container_collect_all - boolean - optional - default: false
+  ## Enable container log collection for all the containers (see ac_exclude to filter out containers)
+  #
+  #  container_collect_all: false
+
+  ## @param logs_dd_url - string - optional
+  ## Define the endpoint and port to hit when using a proxy for logs. The logs are forwarded in TCP
+  ## therefore the proxy must be able to handle TCP connections.
+  #
+  #  logs_dd_url: <ENDPOINT>:<PORT>
+
+  ## @param logs_no_ssl - boolean - optional - default: false
+  ## Disable the SSL encryption. This parameter should only be used when logs are
+  ## forwarded locally to a proxy. It is highly recommended to then handle the SSL encryption
+  ## on the proxy side.
+  #
+  #  logs_no_ssl: false
+
+  ## @param processing_rules - list of custom objects - optional
+  ## Global processing rules that are applied to all the logs. The available rules are
+  ## "exclude_at_match", "include_at_match" and "mask_sequences". More information in Datadog documentation:
+  ## https://docs.datadoghq.com/logs/log_collection/#global-processing-rules
+  #
+  #  processing_rules:
+  #    - type: <RULE_TYPE>
+  #      name: <RULE_NAME>
+  #      pattern: <RULE_PATTERN>
+
+  ## @param use_port_443 - boolean - optional - default: false
+  ## By default, logs are sent to port 10516 (for the US site), use this parameter
+  #  to force the Agent to send logs in TCP to port 443 for the EU site
```

Comment:
Let's remove reference to the EU site here (probably my mistake).

Reply:
this parameter is useless? we have the possibility to set up the Site in this file.

Reply:
This parameter is for US only.
On EU we already send logs through 443, but on US we have a specific parameter to do this.